application/src/main/java/com/github/pieter_groenendijk/model/DigitalProduct.java

```diff
@@ -0,0 +1,55 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "DigitalProduct")
+public class DigitalProduct extends Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long digitalProductId;
+
+    @Column(name = "isAvailable", nullable = false, length = 100)
+    private String isAvailable;
```

Comment:
Je beschrijft in Teams:
Ik heb voor een digitaalProduct een eigenschap (hernoemd) naar isAvailable (kan ook comingSoon zijn ofzo). Dat de beheerder een product kan toevoegen en adverteren, maar nog niet beschikbaar is om uit te streamen. Dat was een gevoelsding, mag ook weg - zegt u het maar.

Ik had dit in eerste instantie ook met een draft, published achtig iets. Echter was toen besproken dat het buiten de requirements viel en dus weggelaten kon worden.

Reply:
Yo, dan knikker ik t er uit. Genoeg dingen te doen!